dubbo-common/pom.xml

```diff
@@ -39,6 +39,10 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Let's avoid of introducing new dependency into the distribution. In this case, it looks like FastDateFormat doesn't help us much, since we are using the same instance in single thread to format all.

If I am wrong, try to include FastDataFormat's source code in Dubbo project, and update LICENSE file.

Reply:
As FasDataFormat has better performance then other as a sample performance check was done by @lixiaojiee  [here](https://github.com/apache/incubator-dubbo/pull/3090#issuecomment-453394557). 

As you suggested we can add the source code of FastDataFormat in dubbo project it self, would you help me by guiding what is the approach of doing it.  Is that we can copy code of FastDataFormat and adding a license file? Any help would be appreciated.

Reply:
@khanimteyaz 
I think u should copy the code of the class. I saw that the common-langs use apache lisence, so don't forget copy the license header (pls use FastDateFormtat's license header not dubbo's).

After copy the code, u should modify dubbo-parent file (pom file). U should add a apache-rat-plugin's exclude line like `<exclude>**/org/apache/dubbo/common/threadlocal/InternalThreadLocalMap.java</exclude>`. 

Finally, u should modify the LICENSE file. In the tail of the file, it can also find some examples.

Reply:
@carryxyh and @beiwei30 thanks for guidance. I have made the requires changes. Could you have a look.